python/node_test.py

```diff
@@ -1,26 +1,30 @@
 # tests that correspond to the `src/doc.rs` rust api
 import tempfile
+from typing import cast
+import iroh.iroh_ffi
 import pytest
 import asyncio
-import iroh
 
-from iroh import Iroh, ShareMode, LiveEventType, AddrInfoOptions, NodeOptions, NodeOptions, NodeAddr, PublicKey
+from iroh.iroh_ffi import Iroh, ShareMode, LiveEventType, AddrInfoOptions, NodeOptions
+# from iroh import Iroh, ShareMode, LiveEventType, AddrInfoOptions, NodeOptions
+# Also works, but most type checkers will not be able to find the type hints
+
 
 @pytest.mark.asyncio
 async def test_basic_sync():
     # setup event loop, to ensure async callbacks work
-    iroh.iroh_ffi.uniffi_set_event_loop(asyncio.get_running_loop())
+    iroh.iroh_ffi.uniffi_set_event_loop(asyncio.get_running_loop())  # type: ignore
 
     options = NodeOptions()
     options.enable_docs = True
 
     # Create node_0
     iroh_dir_0 = tempfile.TemporaryDirectory()
-    node_0 = await Iroh.persistent_with_options(iroh_dir_0.name, options)
+    node_0 = cast(Iroh, await Iroh.persistent_with_options(iroh_dir_0.name, options))
```

Comment:
I don't think we want that

Reply:
That's fair. I included it in a separate commit just in case. Will revert that one.

Reply:
Without it, you don't get type inference on the returned iroh node though...